modules/ROOT/pages/edi-ack-reconciliation.adoc

```diff
@@ -1,19 +1,21 @@
 = EDI Acknowledgment Reconciliation
 
-As your trading partners acknowledge your messages, Partner Manager automatically updates their acknowledgement status. If you want to be notified, you must specifically configure acknowledgement settings when you create the message.
+As your partners acknowledge your messages, Partner Manager automatically updates their acknowledgement status. If you want to be notified, you must specifically configure acknowledgement settings when you create the message.
 
-If you are expecting to receive functional acknowledgements from your trading partners, you must xref:x12-send-settings.adoc[configure the acknowledgement  settings] when creating the outbound message flow.
+If you are expecting to receive functional acknowledgements from your partners, you must xref:x12-send-settings.adoc[configure the acknowledgement  settings] when creating the outbound message flow.
```

Comment:
If you are expecting to receive functional acknowledgments from your partners, you must xref:x12-send-settings.adoc[configure the acknowledgment settings] when creating the outbound message flow.